app/actions/remote/reactions.ts

```diff
@@ -87,23 +88,28 @@ export const removeReaction = async (serverUrl: string, postId: string, emojiNam
         return {error};
     }
 };
-
 export const handleReactionToLatestPost = async (serverUrl: string, emojiName: string, add: boolean, rootId?: string) => {
     try {
-        const {database} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+        const { database } = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
```

Comment:
I expect some lint issues here. Please run `npm run fix` and most of them will be solved automatically.